MovingBlocks/Terasology
# Added snow slabs and snow shoes

### Contains

Added snow shoes and snow slabs (1/8, 1/4, 1/2) for #2406 

### How to test

For snow slabs
1. Open the command console
2. Type in 'give snowslabeight'
3. You will receive the 1/8 snow slab
4. Do the same for (1/4, and 1/2) slabs, by typing 'give snowslabhalf' and 'give snowslabquarter'
5. You will receive the respective snow slabs in your inventory

For snow shoes
1. Open the command console
2. Type in 'give snowshoe'
3. The snow shoe will appear in inventory

### Outstanding before merging

If anything. You can use neat checkboxes! Feel free to delete if not needed

- [ ] Add snowshoe functionality
- [ ] Add stacking for snow slabs
